api/src/main/java/io/strimzi/api/kafka/model/KafkaUserAuthentication.java

```diff
@@ -22,6 +22,7 @@
 @JsonSubTypes({
         @JsonSubTypes.Type(name = KafkaUserTlsClientAuthentication.TYPE_TLS, value = KafkaUserTlsClientAuthentication.class),
         @JsonSubTypes.Type(name = KafkaUserScramSha512ClientAuthentication.TYPE_SCRAM_SHA_512, value = KafkaUserScramSha512ClientAuthentication.class),
+        @JsonSubTypes.Type(name = KafkaUserSaslPlainClientAuthentication.TYPE_SASL_PLAIN, value = KafkaUserSaslPlainClientAuthentication.class),
```

Comment:
Thanks for the advise. There were some issues and I thought it could be related to the user operator so added this part. After the issue resolved I didn't remove this part.
I will make the change to fix this.

Reply:
code updated, please review ..